specification/securityinsights/resource-manager/common/1.0/types.json

```diff
@@ -53,10 +67,6 @@
           "readOnly": true,
           "type": "string"
         },
```

Comment:
>} [](http://example.com/codeflow?start=8&length=1)

Removing etag from the top-level is a breaking change. It looks like this is intentional, so you will need a breaking change review. After that's been sorted, ARM review can continue.

Reply:
Hi @mentat9 ,
I didn't remove an etag, it seems like this because of the way it is shown in the changed files.
Notice that I added a definition named Resource that doesn't contain an etag.
I hope this is clear, let me know if you have any other concerns.